src/device/device-management.ts

```diff
@@ -51,16 +54,30 @@ export type VideoDeviceConstraints = {
 export async function createCameraTrack(
   constraints?: VideoDeviceConstraints
 ): Promise<LocalCameraTrack> {
-  let stream: MediaStream;
+  // let stream: MediaStream;
+  if (isCameraTaskInProgress) {
+    throw new WcmeError(
+      ErrorTypes.CREATE_CAMERA_TRACK_FAILED,
+      `camera track can NOT be created since previous call has not been finished yet`
+    );
+  }
+  if (capturedVideoStream && capturedVideoStream.active) {
```

Comment:
I don't think we can do that - this will throw an error if the app calls createCameraTrack() more than once in order to have multiple video tracks at the same time

Reply:
this is a unsafe behavior if you call the createCameraTrack() more than once before stopping the first one, 
because the constraint params in second getUserMedia call would not take effect which can refer to https://codepen.io/brownzw/pen/KKeEJPx .